### DISCUSSION TITLE
Enterprise billing for Copilot

### DISCUSSION THREAD

**Discussion by smokedlinq** [community/community#19695] [Enterprise] [answered]
Are there any plans to provide enterprise billing for copilot for members of the enterprise?

**Comment by D7EAD** [comment, marked as answer]
Hi!

As of right now, enterprise-managed user accounts aren't yet supported by Copilot. This is likely to change in the near future, as both enterprises and educational institutions have yet to receive individual support for Copilot.

Hopefully this clears things up and, if I've managed to help you, consider **Marking as Answer** so anyone else with this question can more easily find the appropriate answer. Thanks!

**Comment by vikramhn** [comment]
Any update on this, timeline etc?
